src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -258,35 +265,38 @@ uiModules.get('apps/management')
             }, defaultDocTypes);
           }
 
+          function resolveConflicts(objs, { obj }) {
+            const oldIndexId = obj.searchSource.getOwn('index');
+            const newIndexId = objs.find(({ oldId }) => oldId === oldIndexId).newId;
+            if (newIndexId === oldIndexId) {
+              // Skip
```

Comment:
Can we add a few words here to elaborate? e.g. `The index hasn't changed (or whatever conclusion we can draw from this equality check) so we don't need to {do whatever it is we're doing on line 275}.`